config/ember-try.js

```diff
@@ -130,7 +135,9 @@ module.exports = function() {
             },
           },
         },
-      ]
+        embroiderSafe(),
+        embroiderOptimized(),
```

Comment:
These scenarios need to be added to the GH action matrix!

Reply:
Yes! I swear I actually ran `ember-safe` locally too, but then forgot to add it.

I have pushed an update which adds `embroider-safe` to the suite which passes. `embroider-optimized` does not pass, and I've simply left it out of the GH workflow for now.